ruby/ext/google/protobuf_c/upb.h

```diff
@@ -859,7 +859,7 @@ bool upb_strtable_resize(upb_strtable *t, size_t size_lg2, upb_alloc *a);
  *    the program accesses an invalidated iterator.
  *
  * For C++11 range-based for:
- *  - iterators must be copyable
+ *  - iterators must be copiable
```

Comment:
Same thing here about the "copyable" spelling.

Reply:
Done